src/components/MAPI/apps/ClusterDetailAppListWidgetVersion.tsx

```diff
@@ -22,11 +26,20 @@ interface IClusterDetailAppListWidgetVersionProps
   app?: applicationv1alpha1.IApp;
   catalogNamespace?: string | null;
   canListAppCatalogEntries?: boolean;
+  displayUpgradableStatus?: boolean;
+  displayUpstreamVersion?: boolean;
 }
 
 const ClusterDetailAppListWidgetVersion: React.FC<
   React.PropsWithChildren<IClusterDetailAppListWidgetVersionProps>
-> = ({ app, catalogNamespace, canListAppCatalogEntries, ...props }) => {
+> = ({
+  app,
+  catalogNamespace,
+  canListAppCatalogEntries,
+  displayUpgradableStatus = true,
+  displayUpstreamVersion = false,
```

Comment:
Just for my understanding, is the main function of the `displayUpstreamVersion` prop for rendering a variant of this version widget?

Reply:
This is correct